ingestion/src/metadata/ingestion/source/metadata/amundsen.py

```diff
@@ -495,7 +503,31 @@ def get_type_primitive_type(self, data_type):
         return data_type
 
     def get_database_service(self, service_name: str) -> DatabaseService:
-        service = self.metadata.get_by_name(entity=DatabaseService, fqn=service_name)
+        """
+        Method to get and create Database Service
+        """
+        try:
+            file_path = (
+                Path(__file__).cwd() / "examples/amundsen_data/service_type.json"
+            )
+            with open(file_path, encoding="UTF-8") as file:
+                service_type: dict = json.load(file)
+
+        except FileNotFoundError:
+            raise Exception()
+
+        service = self.metadata.create_or_update(
+            CreateDatabaseServiceRequest(
+                name=service_name,
+                displayName=service_name,
+                description="this is code maede",
+                connection=service_type.get(service_name, "None")["connection"],
```

Comment:
this fails when service_name is not found in `service_type` dict as 
`"None"["connections"]` would fail.

also in case we didn't find the service type map maybe we can create a MySQL service and ingest data over there

Reply:
Yes will add it.